.changeset/fair-trains-invite.md

```diff
@@ -0,0 +1,6 @@
+---
+"nextjs-website": minor
+"strapi-cms": minor
```

Comment:
Since something changed on `strapi-cms` as well, I'd create a separate changelog, because I don't think "Adapt to EC version 4.0.0" is meaningful for Strapi